app/views/admin/settings/index.html.erb

```diff
@@ -79,3 +94,15 @@
 
 <p><%= ts("Last updated on %{updated_at} by %{admin_name}.", :updated_at => @admin_setting.updated_at, :admin_name => @admin_setting.last_updated ? @admin_setting.last_updated.login : "---") %></p>
 <!--/content-->
+
+
+<%= content_for :footer_js do %>
```

Comment:
It looks like this isn't actually toggling the form field (I don't see the `toggle_formfield` class), so I think this can be removed. If we are actually toggling the field, though, this should really use the newer `toggleFormField(element_id)` function from application.js here, since as Elz pointed out, we are trying to prefer our centralized JavaScript.

Reply:
Ah yes. I removed the toggle because I couldn't get the check box to both update the parameter and toggle the visibility of the textbox, so I gave up and forgot to remove the Javascript 🤦‍♀️  - I'll try to get to this later in the week! (pls poke me in Slack if no visible action by the weekend)